src/ts/API/index.tsx

```diff
@@ -1,10 +1,11 @@
 const url = (() => {
-  if (typeof location === 'undefined') return 'https://viking-garage-api-prod.herokuapp.com';
+  if (typeof location === 'undefined') return 'https://viking-garage-api-production.herokuapp.com';
   if (location.hostname.match('localhost')) return 'https://viking-garage-api-dev.herokuapp.com';
   if (location.hostname.match('dev')) return 'https://viking-garage-api-dev.herokuapp.com';
   if (location.hostname.match('staging')) return 'https://viking-garage-api-staging.herokuapp.com';
+  if (location.hostname.match('vikinggarage')) return 'https://viking-garage-api-production.herokuapp.com';
```

Comment:
let's be more precise here, the actual hostname is `vikinggarage.com`, only that one should match production API